archive/tar/tar_test.go

```diff
@@ -90,6 +91,19 @@ func TestTarArchive(t *testing.T) {
 				g.Assert(exists("/tmp/extracted/subdir/linkto_test.txt")).IsTrue("failed to create subdir/linkto_test.txt")
 			})
 
+			g.It("Should restore the correct modification timestamp", func() {
+				var err error
+				var file *os.File
+				var fileInfo os.FileInfo
+				for _, element := range mountFiles {
+					file, err = os.Open("/tmp/extracted/" + element.Path)
```

Comment:
Should probably use some of the temporary file capabilities in ioutil for this instead (see https://golang.org/pkg/io/ioutil/#TempDir as an example). For example this wouldn't do what you wanted in the case of windows.

Reply:
While it’s definitely a good idea to have tests work cross-platform, I’m not sure if this PR is the right place to fix this. This project uses Unix-style temp paths throughout the test suite.


    $ grep -ri "/tmp/" . | wc -l
    65

Reply:
Yea I noticed that afterwards. Guess I'll add that as a TODO for myself there.